loki/base/clusterrolebindings/lokistack-gateway-auth-delegator.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: lokistack-gateway-auth-delegator
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:auth-delegator
+subjects:
+  - kind: ServiceAccount
+    name: lokistack-gateway
```

Comment:
Where is this ServiceAccount created?

Reply:
@computate Just wanted to make sure you saw my questions on this PR before this was merged. Perhaps, I should have used the "request changes" button.